test/test_verify.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, kafkaAddress, schemaRegistryAddress, kafkaConnectAddress, cre
 
         self.SEND_INTERVAL = 0.01  # send a record every 10 ms
         self.VERIFY_INTERVAL = 60  # verify every 60 secs
-        self.MAX_RETRY = 120  # max wait time 120 mins
+        self.MAX_RETRY = 5  # max wait time 5 mins
```

Comment:
Changing this to 5 minutes might not pass the 3 hours test. We can fix this in later PRs.

Reply:
Ahh okay, I can change it back and see if it runs successfully.